backend/supervisord.conf

```diff
@@ -46,7 +35,7 @@ stopasgroup=true
 [program:celery_worker_light]
 command=bash -c "celery -A danswer.background.celery.celery_run:celery_app worker \
     --pool=threads \
-    --concurrency=${CELERY_WORKER_LIGHT_CONCURRENCY:-24} \
+    --concurrency=${CELERY_WORKER_LIGHT_CONCURRENCY:-${NUM_INDEXING_WORKERS:-24}} \
```

Comment:
this seems wrong, I think this should be replacing `CELERY_WORKER_INDEXING_CONCURRENCY` below, right?

Reply:
that's right, will fix.